cmd/lightclient/sentinel/pubsub.go

```diff
@@ -126,67 +171,7 @@ func (s *Sentinel) startGossip(prefix string) (err error) {
 	return nil
 }
 
-func subscribeGossipTopic[T proto.Packet](
-	s *Sentinel,
-	topic string,
-	newcodec func(*pubsub.Subscription) proto.SubCodec,
-	_ T,
-	opts ...pubsub.TopicOpt,
-) error {
-	topicHandle, err := s.pubsub.Join(topic, opts...)
-	if err != nil {
-		return fmt.Errorf("failed to begin topic %s subscription, err=%s", topic, err)
-	}
-	if topicHandle == nil {
-		return fmt.Errorf("failed to get topic handle while subscribing")
-	}
-
-	subscription, err := topicHandle.Subscribe()
-	if err != nil {
-		return fmt.Errorf("failed to begin topic %s subscription, err=%s", topic, err)
-	}
-	s.subManager.addTopicSub(topic, topicHandle, subscription)
-	s.subManager.gossipResults[topic] = make(chan interface{})
-
-	log.Info("[Gossip] began subscription", "topic", subscription.Topic())
-	var nothing T
-	go runSubscriptionHandler(s, subscription, newcodec, nothing)
-	return nil
-}
-
-// currySubHandler uses a func(ctx *proto.StreamContext, dat proto.Packet) error to handle a *pubsub.Subscription
-// this allows us to write encoding non specific type safe handler without performance overhead
-func runSubscriptionHandler[T proto.Packet](
-	s *Sentinel,
-	sub *pubsub.Subscription,
-	newcodec func(*pubsub.Subscription) proto.SubCodec,
-	_ T,
-) {
-	sd := newcodec(sub)
-	for {
-		var t T
-		val := t.Clone().(T)
-		ctx, err := sd.Decode(s.ctx, val)
-		if err != nil {
-			log.Warn("fail to decode gossip packet", "err", err, "topic", ctx.Topic, "pkt", reflect.TypeOf(t))
-			continue
-		}
-		if ctx.Msg.ReceivedFrom == s.host.ID() {
-			continue
-		}
-		s.subManager.gossipResults[sub.Topic()] <- val
-		if err != nil {
-			log.Warn("failed handling gossip ", "err", err, "topic", ctx.Topic, "pkt", reflect.TypeOf(t))
-			continue
-		}
-		log.Info("[Gossip] Received Subscription", "topic", ctx.Topic)
-	}
-}
-
 func (s *Sentinel) getTopicByName(name TopicName) string {
-	return fmt.Sprintf("%s/%s/ssz_snappy", s.subManager.prefix, name)
-}
-
-func (s *Sentinel) GossipChannel(topic TopicName) chan interface{} {
-	return s.subManager.gossipResults[s.getTopicByName(topic)]
+	prefix := "/eth2/4a26c58b"
```

Comment:
please compute the fork digest instead of hardcoding it

Reply:
done!